keycloak/keycloak-admin-ui
# Group details

## Motivation
Added the group details to the group section

## Verification Steps

Go to the groups section create a couple of groups click on the group see the new group details

## Checklist:


## Additional Notes
